consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -37,12 +48,26 @@ public void individualRequest(IIndividualRequest individualRequest,
                                   long connectionTimeOut,
                                   long receiveTimeOut) throws OutageException, TimeoutException {
         LOGGER.debug("Invoke cmc target application service for individual request");
+        String sdtRequestReference = individualRequest.getSdtRequestReference();
```

Comment:
The changes made to this class for the claim status update (see #104) added ```String requestType = individualRequest.getRequestType();``` before this line and used it in place of ```individualRequest.getRequestType()``` in the if-elseif statement below.  Might want to do the same here to make the merge easier.

Reply:
we will address this during the merge